docs/service_controller.md

```diff
@@ -0,0 +1,29 @@
+# The Service Controller
+
+The service controller is responsible for watch for service and node object changes, so that it can create, update, or delete cloud load balancers corresponding to load balanced services.  Like the other controllers, we import the cloud-provider provided utility functions for managing the controller itself, which calls into cloud provider defined methods `GetLoadBalancer`, `GetLoadBalancerName`, `EnsureLoadBalancer`, `UpdateLoadBalancer`, and `EnsureLoadBalancerDeleted`.
+
```

Comment:
Annotation values have no constraints, right? I looked at the docs and looks like there are rules only for the keys. Just making sure so we don't give the users the wrong impression with `Valid Values : - `

Reply:
Yeah it might not be quite the correct way to phrase it..